exporters/stdout/metric.go

```diff
@@ -44,13 +45,37 @@ type line struct {
 
 	Quantiles []quantile `json:"Quantiles,omitempty"`
 
+	Histogram histogram `json:"histogram,omitempty"`
+
 	// Note: this is a pointer because omitempty doesn't work when time.IsZero()
 	Timestamp *time.Time `json:"Timestamp,omitempty"`
 }
 
-type quantile struct {
-	Quantile interface{} `json:"Quantile"`
-	Value    interface{} `json:"Value"`
+type (
+	boundary struct {
+		Min float64 `json:"min"`
+		Max float64 `json:"max"`
+	}
+
+	bucket struct {
+		boundary `json:"bounds"`
+		Count    float64 `json:"count"`
+	}
+
+	histogram []bucket
+
+	quantile struct {
+		Quantile interface{} `json:"Quantile"`
+		Value    interface{} `json:"Value"`
+	}
+)
+
+func (hist histogram) MarshalJSON() ([]byte, error) {
+	bytes, err := json.Marshal(hist)
+	if err != nil {
+		return nil, err
+	}
+	return append(bytes, []byte("\n"+printHistogram(hist))...), nil
```

Comment:
I'm not following why we are Marshaling with json and printing the histogram. This is redundantly showing the same information.